components/infobox/wikis/pubgmobile/infobox_map_custom.lua

```diff
@@ -40,63 +33,43 @@ MODES.br = MODES['battle royale']
 ---@param frame Frame
 ---@return Html
 function CustomMap.run(frame)
-	local customMap = Map(frame)
-	customMap.createWidgetInjector = CustomMap.createWidgetInjector
-	customMap.addToLpdb = CustomMap.addToLpdb
-	_args = customMap.args
-	return customMap:createInfobox()
-end
+	local map = CustomMap(frame)
+	map:setWidgetInjector(CustomInjector(map))
 
----@return WidgetInjector
-function CustomMap:createWidgetInjector()
-	return CustomInjector()
+	return map:createInfobox()
 end
 
+--@param id string
 ---@param widgets Widget[]
 ---@return Widget[]
-function CustomInjector:addCustomCells(widgets)
-	table.insert(widgets, Cell{
-		name = 'Span',
-		content = {_args.span}
-	})
-	table.insert(widgets, Cell{
-		name = 'Theme',
-		content = {_args.theme}
-	})
-	table.insert(widgets, Cell{
-		name = 'Size',
-		content = {_args.size}
-	})
-	table.insert(widgets, Cell{
-		name = 'Game Version',
-		content = {CustomMap._getGameVersion()}
-	})
-	table.insert(widgets, Cell{
-		name = 'Game Mode(s)',
-		content = {CustomMap._getGameMode()}
-	})
+function CustomInjector:parse(id, widgets)
+	local args = self.caller.args
+	if id == 'custom' then
+		Array.appendWith(widgets,
+			Cell{name = 'Span', content = {args.span}},
+			Cell{name = 'Theme', content = {args.theme}},
+			Cell{name = 'Size', content = {args.size}},
+			Cell{name = 'Game Version', content = {Game.name{game = self.caller.args.game}}},
```

Comment:
this removes the links etc pp
also the input values of the old module seem to not all exist in the game data (Module:Info)
question would be what inputs we actually have here used on the wiki and how to clean it up

Reply:
It looks like this is a leftover field from when the wikis were still one. This would signify that the maps were for mobile or PC/Console. It should be safe to remove these entirely considering the wiki name is "PUBG Mobile"